functions/utils/analytics.js

```diff
@@ -0,0 +1,87 @@
+const functions = require("firebase-functions");
+const { db, admin } = require("./admin");
+
+/*
+    writeToAnalytics adds an event trigger to 'applicants' collection.
+    The event trigger updates a separate document's fields in the 'analytics' collection.
+    Fields include: 
+                -> total number of applicants
+                -> number of UCSC student applicants
+                -> number of first time hackers
+    **NOTE**
+    Fields are not mutually exclusive!
+    i.e. in the event of a document creation;
+            -> if an applicant were a first time hacker and a UCSC student:
+                --> ALL THREE FIELDS WILL BE INCREMENTED BY ONE
+                    
+*/
+
+const service = functions.firestore.document("applicants/{docId}").onWrite(async (change, context) => {
+  const oldDocument = change.before.exists ? change.before.data() : null;
+  const document = change.after.exists ? change.after.data() : null;
+  const analyticsDoc = db.collection("analytics").doc("applicant-analytics");
+  if (document && !oldDocument) {
+    // onCreate case
+    const toUpdate = {
+      ucscIncrement: document.UCSC_Student ? 1 : 0,
+      firstTimeIncrement: document.First_CruzHack ? 1 : 0,
```

Comment:
In terms of this naming, let's try to avoid using snake casing for field names and try sticking to camel casing.